tap-client/src/components/home/HomeComponent.css

```diff
@@ -0,0 +1,49 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Arial;
+    padding: 20px;
+    background-color: #dddddd;
```

Comment:
Dont use background color unless for very small components. I set a background to the app

Reply:
Corrected to the best of my knowledge